pydantic/schema.py

```diff
@@ -648,6 +667,11 @@ def add_field_type_to_schema(field_type: Any, schema: Dict[str, Any]) -> None:
             break
 
 
+def get_schema_ref(ref_name: str, schema_overrides: bool) -> Dict[str, Any]:
```

Comment:
fix this going to conflict with #1480?

Reply:
Hmm I'll update this PR is #1480 is merged first. But imo it should be quite easy to resolve conflicts

Reply:
What's the benefit of pulling this logic into this function?